src/lib/state/ui/index.ts

```diff
@@ -36,7 +37,7 @@ class Store {
             emojiFont: createPersistentState("uplink.ui.emojiFont", EmojiFont.Fluent),
             theme: createPersistentState("uplink.ui.theme", "default"),
             cssOverride: createPersistentState("uplink.ui.cssOverride", ""),
-            sidebarOpen: createPersistentState("uplink.ui.sidebarOpen", true),
+            sidebarOpen: createPersistentState("uplink.ui.sidebarOpen", !checkMobile()),
```

Comment:
@Flemmli97 It will apply for small width in web as well, is correct?

Reply:
yes